seirmo/slider.py

```diff
@@ -0,0 +1,125 @@
+#
+# This file is part of seirmo (https://github.com/SABS-R3-Epidemiology/seirmo/)
+# which is released under the BSD 3-clause license. See accompanying LICENSE.md
+# for copyright notice and full license details.
+#
+
+import dash_core_components as dcc
+import dash_html_components as html
+import numpy as np
+
+
+class _SliderComponent(object):
+    """Slider Class:
+
+    Controls the creation and grouping of slider for the app
+    """
+
+    def __init__(self): # noqa
+        super(_SliderComponent, self).__init__()
+
+        self._slider_ids = []
```

Comment:
is it necessary to store the slider ids separately from the sliders dict? You can retrieve the slider ids from the dict with `list(self._sliders.keys())`.